src/components/Coaching/CoachingDetail/AppointmentResults/AppointmentResults.tsx

```diff
@@ -98,82 +93,26 @@ export const AppointmentResults: React.FC<AppointmentResultsProps> = ({
             >
               <TableBody>
                 <HeaderRow role="rowheader">
-                  <AlignedTableCell>{t('Appointments')}</AlignedTableCell>
-                  {appointmentResults.map(({ id, startDate }) => (
-                    <AlignedTableCell key={id}>
+                  <AlignedTableCell></AlignedTableCell>
+                  {appointmentResults.map(({ startDate, endDate }) => (
+                    <AlignedTableCell key={startDate}>
                       {startDate &&
-                        dateFormatWithoutYear(
-                          DateTime.fromISO(startDate),
-                          locale,
-                        )}
+                      endDate &&
```

Comment:
This code could use the helper function I recommended extracting from `ActivitySummary`.